tests/UnitTests/CoreGraphics.drawing/DrawingTest.cpp

```diff
@@ -26,6 +26,21 @@
 #import "CGContextInternal.h"
 #endif
 
+#ifdef TARGET_OS_MAC
+#define LOG_TEST_FILE(...)
+#else
+#define LOG_TEST_FILE(...) WEX::Logging::Log::File(__VA_ARGS__)
+std::wstring _CFStringToWString(CFStringRef string) {
+    const UniChar* characters16 = CFStringGetCharactersPtr(string);
+    if (characters16) {
+        return {characters16, characters16 + CFStringGetLength(string)};
+    }
+
+    static std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+    return converter.from_bytes(CFStringGetCStringPtr(string, kCFStringEncodingUTF8));
+}
```

Comment:
add a newline before this guy? I was confused what the log test thing did until I realized this was separate.